sr_system_info/src/sr_system_info/system_info.py

```diff
@@ -15,18 +15,17 @@
 # with this program. If not, see <http://www.gnu.org/licenses/>.
 
 from __future__ import absolute_import
+from builtins import round
+import subprocess
 import argparse
 import dynamic_reconfigure
 import dynamic_reconfigure.client
 import psutil
-import rospy
 import rospkg
-import subprocess
 import yaml
-from builtins import round
 
 
-class SystemInfo(object):
+class SystemInfo:
```

Comment:
Just curious - what's the thinking behind this?

Reply:
From python3 onward, all classes are inherited from object, so it's better not to specify it as there might be a variable, class or whatever named object that is imported from another file